pkg/wallet/api.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package wallet
+
+// Wallet interface
+type Wallet interface {
```

Comment:
You might want to consider defining a few "sub" interfaces and then have the Wallet be a composition of them.

Reply:
I will work into break it into multiple interfaces

Reply:
done